app/src/main/java/com/krachkovsky/it_anews/ui/fragments/NewsArticleFragment.kt

```diff
@@ -0,0 +1,54 @@
+package com.krachkovsky.it_anews.ui.fragments
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.webkit.WebViewClient
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import androidx.navigation.ui.setupWithNavController
+import com.krachkovsky.it_anews.databinding.FragmentArticleBinding
+
+class NewsArticleFragment : Fragment() {
+    private var _binding: FragmentArticleBinding? = null
+    private val binding
+        get() = requireNotNull(_binding) {
+            "View was destroyed"
+        }
+
+    private val args: NewsArticleFragmentArgs by navArgs()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return FragmentArticleBinding.inflate(inflater, container, false)
+            .also { binding ->
+                _binding = binding
+            }
+            .root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        var article = args.url
```

Comment:
почему url это article?
```suggestion
        val article = args.url
```